### PR TITLE
docs: add requirements section

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,31 +26,40 @@
 
 # FingerprintJS Pro Chrome Extension Example
 
+Fingerprint is a device intelligence platform offering 99.5% accurate visitor identification.
+
 This repository contains an example Chrome extension that uses the FingerprintJS FingerprintJS Pro JavaScript agent.
+
+## Requirements
+
+- Chromium based browser (Chrome, Edge, Opera, etc.)
```
Comment (JuroUhlar) [inline README.md:35]:
```suggestion
- Chromium-based browser (Chrome, Edge, Opera, etc.)
```

File: README.md [hunk 2]
```diff
@@ -26,31 +26,40 @@
...
 
-### How it works
+## How it works
 
 The solution is split into two parts - **chrome-extension** and **website**.
 
-#### Website
+### Website
 
 The website uses the FingerprintJS Pro JavaScript agent as it normally would, and communicates with the Chrome extension.
```
Comment (JuroUhlar) [inline README.md:51]:
```suggestion
The website uses the Fingerprint Pro JavaScript agent as it normally would, and communicates with the Chrome extension.
```

File: README.md [hunk 3]
```diff
@@ -26,31 +26,40 @@
...
 It is hosted on [Github Pages](https://fingerprintjs.github.io/fingerprintjs-pro-chrome-extension-example/).
 
-#### Chrome Extension
+### Chrome Extension
 
 Provides sample extension that consists of popup page, background, and content script.
```
Comment (JuroUhlar) [inline README.md:56]:
```suggestion
Provides a sample extension that consists of a popup page, background, and content script.
```

File: README.md [hunk 4]
```diff
@@ -26,31 +26,40 @@
...
+
 You can install the example chrome extension from [Chrome Web Store](https://chrome.google.com/webstore/detail/fingerprintjs-example-bro/knppbjgkegnlbhddedbilnfmnkdocekn).
 
-### Why it exists
+## Why it exists
 
 The Chrome extension environment has limited capabilities for external scripts running within content scripts. This repository showcases different strategies on how FingeprintJS Pro can be used in this ecosystem. 
 
-### How it works
+## How it works
```
Comment (JuroUhlar) [inline README.md:45]: It seems unnecessary to have a whole section for one sentence. Consider combining the two  Why and How sections into one "How it works" section

File: README.md [hunk 5]
```diff
@@ -26,31 +26,40 @@
...
 The website uses the FingerprintJS Pro JavaScript agent as it normally would, and communicates with the Chrome extension.
 It is hosted on [Github Pages](https://fingerprintjs.github.io/fingerprintjs-pro-chrome-extension-example/).
 
-#### Chrome Extension
+### Chrome Extension
 
 Provides sample extension that consists of popup page, background, and content script.
 
-### Strategies
+## Strategies
```
Comment (JuroUhlar) [inline README.md:58]: It seems both these strategies are related to the "Chrome Extension" environment, correct? In that case, the heading level should be lower than "Chrome extension". To make the heading hierarchy work, consider

* Making "Website" and "Chrome extension" into H2s
* Removing "Strategies" H2, just continue the Chrome extension section with "This repository contains..." 
* Now you can keep "New window extension strategy" and "Iframe extension strategy" as H3s and it hierarchically makes sense

File: README.md [hunk 6]
```diff
@@ -26,31 +26,40 @@
 
 # FingerprintJS Pro Chrome Extension Example
```
Comment (JuroUhlar) [inline README.md:27]:
```suggestion
# Fingerprint Pro Chrome Extension Example
```

Please check for outdated FingerprintJS Pro references in the whole file

File: README.md [hunk 7]
```diff
@@ -76,7 +85,7 @@ This strategy can be found in `packages/chrome-extension/src/fingerprint/strateg
 
 **Note:** Implementation of both of these strategies is not perfect, ideally they should also handle time-out scenarios and connection issues with the website. It was decided to leave it out to keep it simple.
 
-### Development
+## Development
 
 After cloning the repository perform these operations:
```
Comment (JuroUhlar) [inline README.md:91]: Add the Support and feedback (optional) and Licence sections to the end of the Readme file.

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (TheUnderScorer) [reply README.md:41]: Done!

File: README.md (hunk 7 above)
Comment (TheUnderScorer) [reply README.md:86]: Done! I've added both sections

File: README.md (hunk 5 above)
Comment (TheUnderScorer) [reply README.md]: Great suggestion, done!

**Review by JuroUhlar — APPROVED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**